### ISSUE TITLE
Description of the function "getNewContract": The usage defined is for RPC and not "aut"

### ISSUE THREAD

**Issue by jfaiza** [autonity/docs.autonity.org#213]
Hello,

On the description of the function "getNewContract": The "Usage" defined is for RPC and not for "aut" :

![image](https://github.com/user-attachments/assets/9aea2bfc-015d-4d8f-b099-b8371b9af052)

**Comment by jfaiza** [comment]
The same problem for the function "getOracle":

![image](https://github.com/user-attachments/assets/de01558b-f27e-439a-8e36-8bb6b078b577)

**Comment by jay-cousins** [comment]
Good catch @jfaiza , thanks for spotting this ! Quite right. Triaging as low, priority 3, to be fixed in next iteration of docs.
